src/libs/SidebarUtils.ts

```diff
@@ -278,7 +278,8 @@ function getOptionData({
         hasParentAccess: undefined,
         isIOUReportOwner: null,
         isChatRoom: false,
-        isArchivedRoom: false,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        private_isArchived: undefined,
```

Comment:
This `result` object type is `ReportUtils.OptionData`, but `ReportUtils.OptionData` still has element `isArchivedRoom?: boolean | null;`, not `private_isArchived`.